compiler/passes/codegen.cpp

```diff
@@ -530,6 +530,67 @@ static void codegen_header_compilation_config() {
 }
 
 
+static void protectNameFromC(Symbol* sym) {
+  //
+  // Let's assume we only have to rename user symbols and that we've
+  // done a good job of protecting our internal and standard module
+  // symbols.  This has the advantage of not having to take special
+  // pains to keep from renaming things like chpl_string and uint64_t
+  // (which perhaps should arguably also have FLAG_EXTERN, but don't
+  // consistently seem to today; and adding FLAG_EXTERN to them seems
+  // to result in it getting propagated to type aliases and such in
+  // ways that caused headaches for me).
+  //
```

Comment:
I don't have a strong preference. I think this is a positive step forward and that most conflicts will come from user code anyways.

Up to you how you want to proceed. Personally, I would do it before committing. Needing to make the change to commit would be a good forcing function for me, but if I planned to make the improvement later I think I'd end up pushing it to the back burner pretty easily.